src/getexchange.c

```diff
@@ -258,7 +258,7 @@ int getexchange(void) {
 
 	    case KEY_F(2) ... KEY_F(11): {
 		/* F2...F11 - F1 = 1...10 */
-		send_standard_message(x - KEY_F(1));
+                send_standard_message(x - KEY_F(1));
```

Comment:
no check for empty call field here?

could the check be moved to a more central place?

Reply:
There were concerns how applicable this feature will be for general public. Overall it seems reasonable to limit when this feature kicks in.
Usually this feature is useful right after you press enter in S&P mode at the end of the line. It means you are in the callsign field. If you moved outside of call sign field it means you are doing something else. We can always expand when this feature kicks in if people request it.

Reply:
'_' is also implemented only in callinput file and not in getexchange.

Reply:
True. But ideally the message keys should work the same way in all input fields. This is unfortunately not the case as there is no common handling for them (yet). In my view we shall not increase this divergence.

As an extra field there is the keyer (Ctrl-K) window where it is quite possible that the user wants to mix F-keys with typed text.

Reply:
I made all three fields: exchange, call input, and keyer behave the same with respect to resend keys(F1..F11, '_'). I'm happy to work on combining all of them into a single function, but maybe in a different PR? They look very different and it will take time to make sure that nothing is missed.

Reply:
Yes, that will require another PR for sure. There is still surely a lot to be fixed or improved.